### PR TITLE
fix: promote "protobufjs" to a dependency from a devDependency

### PR DESCRIPTION
**Pull request by Robin-Hoodie**
Using Yarn 2 in PnP mode will break this package as it seems to require "protobufjs" in runtime code, while it's not specified as a dependency, resulting in 

> "@google-cloud/firestore tried to access protobufjs, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound"

I did not open an issue for this beforehand as the problem it's trying to solve seems obvious

You can view more info on this here: https://yarnpkg.com/configuration/yarnrc#packageExtensions and here https://yarnpkg.com/features/pnp

### CODE REVIEW

**Review by JustinBeckwith — APPROVED** [review]

**Comment by Robin-Hoodie** [comment]
I'm not sure what the last 2 CI tests are being blocked by. Is there anything that's expected from me before being able to merge this PR?

**Comment by schmidt-sebastian** [comment]
We have to kick off the tests on your behalf. They are running now.

**Pull request merged**
